api/migrations/20250402143550_create_changes.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TYPE change_type AS ENUM (
+    'PACKAGE_VERSION_ADDED',
+    'PACKAGE_TAG_ADDED'
+);
+
+CREATE TABLE changes (
+    seq BIGSERIAL PRIMARY KEY,
+    change_type change_type NOT NULL,
+    package_id VARCHAR(255) NOT NULL,
```

Comment:
Seperate `scope_name` and `package_name`.

Reply:
updated 😄